Compute word error rate

lfelipeparra opened the pull request:
-compute word error rate.
-delete id_recording and path_recording field in the SpeechEx model.
-save transcription and word error rate in the database.

lfelipeparra committed:
-add transcription and wer field in the SpeechEx model
-compute word error rate.
-delete id_recording and path_recording field in the SpeechEx model.
-save transcription and word error rate in the database.

patientzero reviewed (changes requested):
Do not check in local paths. 
Make Kaldi root configurable
INTEL_ROOT && KALDI_ROOT in 
settings.local
settings.staging
settings.prod
ALSO: check if kaldi is there, if not, don't run kaldi tasks and write to log, no kaldi is available

lfelipeparra committed:
-add transcription and wer field in the SpeechEx model
-compute word error rate.
-delete id_recording and path_recording field in the SpeechEx model.
-save transcription and word error rate in the database.

lfelipeparra committed:
Merge branch 'master' of https://github.com/patientzero/sma_rest into compute-wer

 Conflicts:
	api/models.py
	api/views.py
	sma_rest/settings.py
	sma_rest/settings/prod.py
	sma_rest/settings/staging.py

lfelipeparra committed:
Merge branch 'master' of https://github.com/patientzero/sma_rest into compute-wer

lfelipeparra committed:
-add transcription and wer field in the SpeechEx model
-compute word error rate.
-delete id_recording and path_recording field in the SpeechEx model.
-save transcription and word error rate in the database.

lfelipeparra committed:
-add transcription and wer field in the SpeechEx model
-compute word error rate.
-delete id_recording and path_recording field in the SpeechEx model.
-save transcription and word error rate in the database.
